### PR TITLE
Extend Katib documentation to use not in GKE

### PR DESCRIPTION
**Pull request by andreyvelich**
I added some information inside Katib package README. 
How to deploy Katib not in GKE if you don't have StorageClass inside your cluster.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2260)

### CODE REVIEW

**Comment by andreyvelich** [comment]
@jlewi @YujiOshima 
Please have a look on this PR.

File: kubeflow/katib/README.md [hunk 1]
```diff
@@ -16,27 +16,28 @@
 
 ## Quickstart
 
-For running Katib you have to install tf-job operator and pytorch operator package.
+For running Katib, you have to install tfjob operator and pytorch operator package.
```
Comment (richardsliu) [inline kubeflow/katib/README.md]: Maybe just say TF operator to be consistent with the repo name.

File: kubeflow/katib/README.md [hunk 2]
```diff
@@ -16,27 +16,28 @@
...
 In your Ksonnet app root, run the following
 
 ```
 export KF_ENV=default
+ks env set ${KF_ENV} --namespace=kubeflow
 ks registry add kubeflow github.com/kubeflow/kubeflow/tree/master/kubeflow
 ```
 
-### TF-job operator
+### TFjob operator
```
Comment (richardsliu) [inline kubeflow/katib/README.md]: Same here - TF operator

File: kubeflow/katib/README.md [hunk 3]
```diff
@@ -55,6 +56,33 @@ ks generate katib katib
 ks apply ${KF_ENV} -c katib
 ```
 
+If you want to use Katib not in GKE and you don't have StorageClass for dynamic volume provisioning at your cluster, you have to create persistent volume to bound your persistent volume claim.
```
Comment (richardsliu) [inline kubeflow/katib/README.md]: Include a link to k8s documentation: https://kubernetes.io/docs/concepts/storage/persistent-volumes/

File: kubeflow/katib/README.md [hunk 4]
```diff
@@ -65,5 +93,11 @@ ks delete ${KF_ENV} -c pytorch-operator
 ks delete ${KF_ENV} -c tf-job-operator
 ```
 
+If you create pv for Katib, delete it
```
Comment (richardsliu) [inline kubeflow/katib/README.md]: Need to delete the pvc as well?

**Pull request merged**